src/allotropy/parsers/appbio_absolute_q/appbio_absolute_q_parser.py

```diff
@@ -19,6 +19,6 @@ def release_state(self) -> ReleaseState:
         return ReleaseState.RECOMMENDED
 
     def to_allotrope(self, named_file_contents: NamedFileContents) -> Model:
-        data_frame = read_csv(named_file_contents.contents, parse_dates=["Date"])
+        data_frame = read_csv(named_file_contents.contents)
```

Comment:
Note: this isn't needed because we parse dates outside of pandas, and fails if "Date" column is missing